packages/workspace/src/schematics/tao-new/schema.json

```diff
@@ -76,6 +76,11 @@
     "appName": {
       "type": "string",
       "description": "Application name."
+    },
+    "root": {
```

Comment:
I'd make it more unique, cause we don't expect the user to provide it. Something like: nx-workspace-root